README.md

```diff
@@ -1,69 +1,41 @@
-![onlinebookstore](https://user-images.githubusercontent.com/34605595/137615096-8447d32d-bddc-4f13-a8ed-3c0f4dd5e04e.png)
+A userfriendly Online Bookstore System project where users can login or register, view the available books, select books along with their quantity, and buy them. Users can also get payment receipt after sucessful payment. The project can also be used by the administrator, who can add new books, remove books, increase and decrease the quantity of books, change the price of the books as well as maintain the selling history of books.
 
-#### To View live demo of this website visit <a href="https://theonlinebookstore.herokuapp.com/" target="_blank">https://theonlinebookstore.herokuapp.com</a><br>
-<span style="color:blue">**This Website is built for following purpose:-**</span>
-- For Selling books online.
-- Maintaining books selling history.
-- Adding and managing books.
-- User Friendly.
-- For Implementation of Generic Servlets in Java.
-- This is a Mini-project developed using Java, Jdbc, And Generic Servlets.
-
-<span style="color:blue">**Admin Have Following Access for this online store site:-**</span>
-- Add New Books.
-- View Books Available.
-- Remove Books.
-- Increase Books Amount.
-
-<span style="color:blue">**Users Have Following Access for this online store site:-**</span>
-- Create New Account or Register.
-- Login.
-- View Available Books.
-- Select Books to Buy.
-- Select Books Quantity.
-- Buy Books.
-- Get Payment Receipt.
+![onlinebookstore](https://user-images.githubusercontent.com/34605595/137615096-8447d32d-bddc-4f13-a8ed-3c0f4dd5e04e.png)
 
 ### Technologies used:-
 1. Front-End Development:
 - Html 5.
-- Css.
+- CSS.
 - Javascript.
-- <a href="https://theonlinebookstore.herokuapp.com" target="_blank">Click Here for Front End View</a>
 
-2. Back-End Development
-- Java [JDK 8+]
+2. Back-End Development:
+- Java
 - JDBC
 - Servlet
-- MySQL
-- Apache Maven
 
-3. Database used.
+3. Database:
 - MySql
 
-### ==== Software And Tools Required ====
-- MySQL
+### ================ Software And Tools Required ================
 - Eclipse [Enterprise Edition]
 - Java [JDK 8+]
 - Tomcat v8.0+
 - Apache Maven
+- MySQL
 
-Note:- This is a Sample Project, So we have used only Generic Servlet and not taken care for Security.
-
-### =============== Dummy Database Initialization =====================
+### ================= Dummy Database Initialization =================
 
 STEP 1: Open MySQL Command Prompt or MySQL Workbench
 
 STEP 2: Login to the administrator user as : ```mysql -u <username> -p``` (Enter Password if asked)
 
-STEP 3 :Copy paste the following MySql Commands:
+STEP 3: Copy paste the following MySql Commands-
 ```MySQL
 create database if not exists onlinebookstore;
 
```

Comment:
The database query seems to be outdated with current command in readme. Please check it once